docs/intro.asciidoc

```diff
@@ -0,0 +1,54 @@
+////
+Goal of this doc:
+Provide all the information a user needs to determine if the product is a good enough fit for their use case to merit further exploration
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[intro]]
+== Elastic Distribution for OpenTelemetry Java
+
+++++
+<titleabbrev>Introduction</titleabbrev>
+++++
+
+:language: Java
+:language_lc: java
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+✅ Intro
+////
+The {distro_name} ("the distro") is a {language} package that provides:
+
+* An easy way to instrument your application with OpenTelemetry.
+* Configuration defaults for best usage.
+
+////
+✅ What is it?
+✅ Why use it?
+////
+A _distribution_ is a wrapper around an upstream OpenTelemetry repository with some customizations. The {distro_name} is a wrapper around the https://opentelemetry.io/docs/languages/{language_lc}[OpenTelemetry SDK for {language}] and includes the following customizations:
+
+* Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
+* Preconfigures the collection of tracing and metrics signals, applying some opinionated defaults, such as which sources are collected by default.
+* Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default.
+
+NOTE: For more details about OpenTelemetry distributions in general, visit the https://opentelemetry.io/docs/concepts/distributions[OpenTelemetry documentation].
```

Comment:
My intention with these bullet points is to explain what differentiates the Elastic distro from [OpenTelemetry SDK forJava](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node) and why a user should choose the Elastic distro. I tried to standardize across languages, but the Node and .NET teams came up with different points.

These following points are used in the Node distro docs. Are one or more also true for the Java distro?

>* Access to SDK improvements and bug fixes contributed by the Elastic team _before_ the changes are available upstream in OpenTelemetry repositories.
>* A single package that includes several OpenTelemetry packages as dependencies, so you only need to install and update a single package (for most use cases).

These following points are used in the .NET distro docs. Are one or more also true for the Java distro?

>* Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
>* Preconfigures the collection of tracing and metrics signals, applying some opinionated defaults, such as which sources are collected by default.
>* Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default.

Is there anything else the Elastic distro adds to the OpenTelemetry SDK for Java that we should highlight here?